run

```diff
@@ -50,10 +50,12 @@ FW_CONFIG_STC="$(parse_config 'STC')"
 FW_CONFIG_BET="$(parse_config 'BET')"
 FW_CONFIG_FWHM="$(parse_config 'FWHM')"
 FW_CONFIG_INT_NORM="$(parse_config 'INT_NORM')"
-FW_CONFIG_HPF_CUTOF="$(parse_config 'HPF_CUTOF')"
+FW_CONFIG_HPF_CUTOFF="$(parse_config 'HPF_CUTOFF')"
 FW_CONFIG_HPF="$(parse_config 'HPF')"
 FW_CONFIG_REG_STANDARD_IMAGE="$(parse_config 'REG_STANDARD_IMAGE')"
 FW_CONFIG_LPF="$(parse_config 'LPF')"
+FW_CONFIG_STD_TEMPLATE="$(parse_config 'STANDARD_TEMPLATE')"
```

Comment:
I'm not sure you can treat inputs like this...you'd have to look at the way Michael does it here: https://github.com/flywheel-apps/hcp-icafix/blob/master/run#L31

Bash gears are complicated...

Reply:
Oh wow. I think I might have picked up the pattern from ICAFix, but that is super weird.